crates/ln-dlc-node/src/ln/app_event_handler.rs

```diff
@@ -234,12 +251,66 @@ where
                 payment_hash,
                 onion_fields: _,
                 amount_msat,
-                counterparty_skimmed_fee_msat: _,
+                counterparty_skimmed_fee_msat,
                 purpose,
                 via_channel_id: _,
-                via_user_channel_id: _,
+                via_user_channel_id,
                 claim_deadline: _,
             } => {
+                if counterparty_skimmed_fee_msat > 0 {
```

Comment:
🔧 A [`match` guard](https://doc.rust-lang.org/rust-by-example/flow_control/match/guard.html) would be more idiomatic and clearer, and you would avoid some indentation.